docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -322,450 +405,821 @@ Storm supports a set of extended comparison operators (comparators) for speciali
 - `Lift by Set Membership (*in=)`_
 - `Lift by Proximity (*near=)`_
 - `Lift by (Arrays) (*[ ])`_
-- `Lift by Tag (#)`_
-- `Recursive Tag Lift (##)`_
 
+Just as with standard comparison operators, lifting by property value with extended comparison operators
+requires:
+
+- the form name or full property name (i.e., the combined form and property name) that you will use to lift
+  the node(s);
+- a :ref:`gloss-comp-operator` to specify how the property value should be evaluated; and
+- the property value.
+
+Each extended comparison operator can be used with any kind of property (primary, secondary, universal, or
+extended) whose :ref:`gloss-type` is appropriate for the comparison used.
 
 .. _lift-regex:
 
 Lift by Regular Expression (~=)
 +++++++++++++++++++++++++++++++
 
-The extended comparator ``~=`` is used to lift nodes based on standard regular expressions.
+The extended comparator ``~=`` is used to lift nodes based on PCRE-compatible regular expressions.
 
-.. NOTE::
-  `Lift by Prefix (^=)`_ is supported for string types and can be used to match the beginning of string properties.
+.. TIP::
+  
+  `Lift by Prefix (^=)`_ can be used to match the beginning of string-based properties.
```

Comment:
Should also note that `^=` is much faster and should be preferred to regex if possible.